### PR TITLE
Add contribution guidelines

### PR DESCRIPTION
**Pull request by digigarlab**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at [{PROJECT_NAME}](mailto:{PROJECT_NAME}@{ORGANISATION_NAME}). All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (dblane-digicatapult) [inline CODE_OF_CONDUCT.md]: Needs a proper email address and etc.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+
+- [Git Commit Messages](#git-commit-messages)
+
+[Additional Notes](#additional-notes)
+
+- [Issue and Pull Request Labels](#issue-and-pull-request-labels)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Digital Catapult Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [cpn-pdr-api@digicatapult.org.uk](mailto:cpn-pdr-api@digicatapult.org.uk).
```
Comment (dblane-digicatapult) [inline CONTRIBUTING.md:35]: Does this email address exist?

**Review by dblane-digicatapult — APPROVED** [review]
Looks good, bear in mind we'll change this org to digicatapult one at some point and please ensure the project email addresses exist.

File: CONTRIBUTING.md (hunk 2 above)
Comment (digigarlab) [reply CONTRIBUTING.md:35]: Nope

**Review by dblane-digicatapult — APPROVED** [review]

**Pull request merged**